### PR TITLE
Add Kubecost links and minor update to the Kubecost contents format

### PR DESCRIPTION
**Pull request by linhlam-kc**
Signed-off-by: Linh Lam <linh@kubecost.com>

*Description of changes:*
Add Kubecost links and minor update to the Kubecost contents format

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -67,13 +67,28 @@ url = "https://eksworkshop.com/"
...
+
+[[menu.shortcuts]]
+name = "<i class='fas fa-chalkboard-teacher'></i> Video - EKS Cost Monitoring with Kubecost"
+url = "https://www.youtube.com/watch?v=-mDPV1huzBY"
+weight = 23
 
 [[menu.shortcuts]]
-name = "<i class='fas fa-chalkboard-teacher'></i> App Mesh Workshop"
-url = "https://www.appmeshworkshop.com/"
+name = "<i class='fas fa-chalkboard-teacher'></i> Video - Using Kubecost to monitor your Kubernetes and infrastructure spend"
```
Comment (youngjeong46) [inline config.toml]: Let's put something like: "Containers from the Couch episode on Kubecost"

File: config.toml [hunk 2]
```diff
@@ -67,13 +67,28 @@ url = "https://eksworkshop.com/"
...
+url = "https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring"
+weight = 23
+
+[[menu.shortcuts]]
+name = "<i class='fas fa-chalkboard-teacher'></i> AWS Cloud billing integration"
+url = "https://docs.kubecost.com/install-and-configure/install/cloud-integration/aws-cloud-integrations"
+weight = 23
+
+[[menu.shortcuts]]
+name = "<i class='fas fa-chalkboard-teacher'></i> Video - EKS Cost Monitoring with Kubecost"
```
Comment (youngjeong46) [inline config.toml]: do we need both Containers from the couch episodes? We could probably just have the latest one covering EKS integrations.

**Review by youngjeong46 — COMMENTED** [review]
Left you a couple comments.

File: config.toml (hunk 1 above)
Comment (linhlam-kc) [reply config.toml]: Good idea ! I will update it.

File: config.toml (hunk 2 above)
Comment (linhlam-kc) [reply config.toml]: I think it is good to have both of them as the first one also cover Kubecost API, Kubecost CLI tool in depth which could also benefit AWS customers

Comment (linhlam-kc) [reply config.toml]: @youngjeong46 I updated the video description and remove the 2nd link from CFTC. We can add it later if it is necessary.

**Review by youngjeong46 — APPROVED** [review]
LGTM.

**Pull request merged**